manifests/dispatch_router.pp

```diff
@@ -0,0 +1,74 @@
+# == Class: capsule::dispatch_router
+#
+# Install and configure Qpid Dispatch Router
+#
+class capsule::dispatch_router (
+) {
+
+  class { 'qpid::router': }
+
+  # SSL Certificate Configuration
+  class { 'certs::qpid_router':
+    require => Package[$qpid::router::router_packages],
```

Comment:
I don't think I can't anchor that to happen before because of puppet-concat's voodoo.

`Class['qpid::config']` contains the actual file entry for the concat bits, and the certs need to be done before the qdrouterd.conf is laid down.